src/file-system-cache-blob-storage.js

```diff
@@ -0,0 +1,93 @@
+var fs = require('fs-plus')
+var path = require('path')
+
+module.exports = (function () {
+  FileSystemCacheBlobStorage.load = function (directory) {
```

Comment:
Yeah, I like that. It implements the `BlobStore` interface using the file-system.